current/runtime/fees.md

```diff
@@ -4,118 +4,151 @@ lang: en
+
+The sum of the three transaction fee components (i.e. `base_fee + length_fee + weight_fee`) is known
+as the _inclusion fee_. Even if a transaction fails, the signer must pay the inclusion fee.
+
+## Default Weight Fees
+
+The default fee system uses _fixed_ weights, which means that the arguments to a dispatch do not
+affect its weight. The tiers in this system are represented by the following enum:
 
 ```rust
 pub enum SimpleDispatchInfo {
-    /// A normal dispatch with fixed weight.
     FixedNormal(Weight),
-    /// A normal dispatch with the maximum weight.
     MaxNormal,
-    /// A normal dispatch with no weight.
-    FreeNormal,
-    /// An operational dispatch with fixed weight.
+    InsecureFreeNormal,
     FixedOperational(Weight),
-    /// An operational dispatch with the maximum weight.
     MaxOperational,
-    /// An operational dispatch with no weight.
-    FreeOperational,
+    FixedMandatory(Weight),
 }
 ```
 
-This enum groups all dispatches into _normal_ or _operational_ and gives them a fixed weight. This
-means that the arguments of a dispatchable function do not affect the weight. A dispatch classified
-as _operational_ is exempt from paying both the `base_fee` and the `length_fee`.
-
-### Free Normal
-
-This means that this function has no weight. It will not contribute to block fullness at all, and no
-weight-fee is applied.
+In order to delineate their purposes, the enums in this group are separated into _dispatch classes_,
+which are also defined by an enum:
 
 ```rust
-#[weight = SimpleDispatchInfo::FreeNormal]
-pub fn some_normal_function_light() { noop(); }
+pub enum DispatchClass {
+    Normal,
+    Operational,
+    Mandatory,
+}
 ```
 
-### Fixed Normal
+### Normal Dispatches
 
-This function will always have a weight `10`.
+Dispatches in this class represent normal user-triggered transactions. These types of dispatches may
+only consume a portion of a block's total weight limit; this portion can be found by examining the
+[`AvailableBlockRatio`](https://substrate.dev/rustdocs/master/frame_system/trait.Trait.html#associatedtype.AvailableBlockRatio).
+Normal dispatches are assigned a priority that is proportional to their weight.
+
+#### `FixedNormal`
+
+Describes a normal function that will always have the specified weight.
 
 ```rust
 #[weight = SimpleDispatchInfo::FixedNormal(10)]
 pub fn some_normal_function_heavy() { some_computation(); }
 ```
 
-### Fixed Operational
+#### `MaxNormal`
+
+This is equivalent to `SimpleDispatchInfo::FixedNormal(Weight::max_value())`.
 
-This function will have a fixed weight but can consume the reserved operational portion as well.
+#### `InsecureFreeNormal`
+
+This means that the function has no weight; it will not contribute to block fullness at all and no
+weight-fee is applied. Although the `base_fee` and `length_fee` still need to be paid, as the name
+indicates a lack of weight fees also implies a lack of security, so dispatches of this type should
+be used with care.
```

Comment:
I think the verb "used" here is a little vague. One reason to always try writing in the active voice.
```suggestion
indicates a lack of weight fees also implies a lack of security, so developers should use care
when adding dispatches of this type to their runtime.
```